nix/build/hacks.nix

```diff
@@ -14,7 +14,7 @@ let
     src = builtins.fetchTarball {
       url = "https://gitlab.com/dannywillems/rustc-bls12-381/-/archive/0.7.2/rustc-bls12-381-0.7.2.tar.gz";
     };
-    cargoSha256 = "1wdapzy2qk7ml17sihls3pykj740spzrm8mbvh4495wq5r07v2gr";
+    cargoSha256 = "12vpmrvz91ysjgliyibc3b0fliw9wh8s0j7amdnq1nbr79c8vpxs";
```

Comment:
Yeah, it was breaking builds for me locally (I guess the CI machines had it cached) for months before I realised what was going on :confused: I guess it might be related to the same hash-related fiasco that we went through in July or so, when we were still using tezos from opam